### DISCUSSION TITLE
After using the tool, the keyboard that comes with my laptop stopped working r7000 r7 4800h

### DISCUSSION THREAD

**Discussion by TangNPC** [DavidS95/Smokeless_UMAF#63] [Q&A]
please help me 😭
Is there any solution?
 I have flashed the official Lenovo bios, but my keyboard still does not work.

**Comment by wearver** [comment]
Have you already found a solution? What was the last change you made?

Updating the bios won't solve it, I don't know exactly where the parameters are registered, but I had a problem that didn't appear in the bios, I only solved it by making some changes in the menu itself... updating the bios didn't solve my problem either...

**Reply by TangNPC** [reply]
My solution is that I no longer care whether the keyboard can be used or not, because even if I care, I can't find a solution.

**Reply by TangNPC** [reply]
If you find a solution, let me know，plz

**Reply by TangNPC** [reply]
Hey buddy, I installed this tool and restarted my computer, my keyboard seems to work, you can try it
Hope this helps
https://github.com/dorssel/usbipd-win/releases
